File: Airline/src/java/airline/model/Ciudad.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package airline.model;

/**
 *
 * @author kerly
 */
public class Ciudad implements Jsonable {

    public Ciudad(String codigo, String nombre, String pais) {
        this.codigo = codigo;
        this.nombre = nombre;
        this.pais = pais;

    }
    
    public Ciudad() {
        this.codigo = "";
        this.nombre = "";
        this.pais = "";
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getPais() {
        return pais;
    }

    public void setPais(String pais) {
        this.pais = pais;
    }
    
    String codigo;
    String nombre;
    String pais;

    
}

File: Airline/web/js/Ciudad.js

/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

function Ciudad(codigo, nombre, pais){
    this.Ciudad(codigo, nombre, pais);
}

Ciudad.prototype={
    Ciudad: function(codigo, nombre, pais){
        this.codigo=codigo;
        this.nombre=nombre;
        this.pais=pais;
    }
};


